games-util/pyfa/pyfa-2.19.0.ebuild

```diff
@@ -0,0 +1,100 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+PYTHON_COMPAT=( python3_6 )
```

Comment:
You should really start looking into python3_7 and 3_8 support here.

Reply:
Yeah I actually check for that with every release. There are still ebuilds the package depends on that don't support 3.7 and 3.8.

What is the process here? Should I change those ebuilds and make separate pull requests for them? Or should I simply wait until the maintainers of those packages change them?

Reply:
Yeah, normally it'd be nice if you tested it and made a pull request, but python team has been quite bad at merging PRs lately. 

Can you give me a list of deps that need python_compat updates, and I'll check what I can do?

Reply:
Here is the list:
- dev-python/logbook
- dev-python/markdown2
- dev-python/roman
- dev-python/matplotlib (3.7 but not 3.8)

This are only the direct dependencies. I check them by hand. Is there any tool that might help here?

Reply:
`pkgcheck` comes to mind...

Reply:
logbook and markdown2 seem to have some failing tests. If you can work on getting these to work it'd be awesome. 
roman now has 3.7 and 3.8 support, matplotlib has 3.7 support and lots of dependencies missing 3.8. So at this time it's not wise to focus on matplotlib.

But do you have time/interest looking at logbook and markdown2? It's important that test phases run.

I see Fedora and Debian don't run tests either but support python3.8 for them... but we'd then need some runtime testing done.

Reply:
@juippis ok, will take a look at logbook and markdown2 and make pull request once I'm able to fix them.

Reply:
Thanks. Ping me in the PR when you get them done, because they get assigned to a team I'm not part of.

You'd also benefit by having a bug opened, similarily to this:
https://bugs.gentoo.org/718398

so other people could see and perhaps work on them. I'd also work as a tracker if they are fixed or not if you don't have time to dig into it immediately.